data/workflows/en/01_bootstrap_project.md

```diff
@@ -39,7 +44,6 @@ dune exec bin/main.exe
 
 Or you may be looking for the best way to get started with a specific kind of project:
 
-- A library
```

Comment:
What was the reason for removing this? The other bullet points are more end-user applications (executables) -- do you think we need something more specific?

Reply:
ya, sure thing. my thoughts are as follows:

- the content above discuss using `dune init`
- the content shows `dune init library NAME PATH` from the `--help` dump

to state later in the document "if you need a library ..." right after mentioning tooling that can setup a library i perceive as conflicting information.

counter point, the examples above used the `project` `kind` for `dune init`, so perhaps it's not super explicit. nonetheless, the `dune init` flow or a library kind is capable.